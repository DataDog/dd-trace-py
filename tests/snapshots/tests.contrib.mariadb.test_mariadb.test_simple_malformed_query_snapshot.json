[[
  {
    "name": "mariadb.query",
    "service": "mariadb",
    "resource": "SELEC 1",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "sql",
    "error": 1,
    "meta": {
      "db.name": "test",
      "db.user": "test",
      "error.msg": "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'SELEC 1' at line 1",
      "error.stack": "Traceback (most recent call last):\n  File \"/root/project/ddtrace/contrib/dbapi/__init__.py\", line 75, in _trace_method\n    return method(*args, **kwargs)\nmariadb.ProgrammingError: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SELEC 1' at line 1\n",
      "error.type": "mariadb.ProgrammingError",
      "out.host": "127.0.0.1",
      "runtime-id": "afb83c6e2adf41f9a723786fdb1b223b"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.rowcount": -1,
      "out.port": 3306,
      "system.pid": 5857
    },
    "duration": 1061000,
    "start": 1633039294872805000
  }]]
