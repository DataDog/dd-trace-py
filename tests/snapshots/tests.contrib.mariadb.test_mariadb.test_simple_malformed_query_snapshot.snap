[[{"name" "mariadb.query"
   "service" "mariadb"
   "resource" "SELEC 1"
   "type" "sql"
   "error" 1
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1626718453234358000
   "duration" 3750000
   "meta" {"runtime-id" "0cbfb96da3ee4e86be3cf3b8a4004f86"
           "out.host" "127.0.0.1"
           "db.user" "test"
           "db.name" "test"
           "error.msg" "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'SELEC 1' at line 1"
           "error.type" "mariadb.ProgrammingError"
           "error.stack" "Traceback (most recent call last):\n  File \"/root/project/ddtrace/contrib/dbapi/__init__.py\", line 73, in _trace_method\n    return method(*args, **kwargs)\nmariadb.ProgrammingError: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'SELEC 1' at line 1\n"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 1502
              "out.port" 3306
              "db.rowcount" -1
              "_dd.tracer_kr" 1.0}}]]
